package/MDAnalysis/coordinates/ParmEd.py

```diff
@@ -20,336 +20,14 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
-
-"""ParmEd structure I/O --- :mod:`MDAnalysis.coordinates.ParmEd`
-================================================================
-
-Read coordinates data from a `ParmEd <https://parmed.github.io/ParmEd/html>`_ :class:`parmed.structure.Structure` with :class:`ParmEdReader` 
-into a MDAnalysis Universe. Convert it back to a :class:`parmed.structure.Structure` with 
-:class:`ParmEdConverter`.
-
-Example
--------
-
-ParmEd has some neat functions. One such is `HMassRepartition`_. 
-This function changes the mass of the hydrogens in your system to your 
-desired value. It then adjusts the mass of the atom to which it is 
-bonded by the same amount, so that the total mass is unchanged. ::
-
-    >>> import MDAnalysis as mda
-    >>> from MDAnalysis.tests.datafiles import PRM
-    >>> u = mda.Universe(PRM)
-    >>> u.atoms.masses[:10]
-    array([14.01 ,  1.008,  1.008,  1.008, 12.01 ,  1.008, 12.01 ,  1.008,
-        1.008,  1.008])
-
-We can convert our universe to a ParmEd structure to change our hydrogen masses. ::
-
-    >>> import parmed.tools as pmt
-    >>> parm = u.atoms.convert_to('PARMED')
-    >>> hmass = pmt.HMassRepartition(parm, 5)  # convert to 5 daltons
-    >>> hmass.execute()
-
-We can then convert it back to an MDAnalysis Universe for further analysis. ::
-
-    >>> u2 = mda.Universe(parm)
-    >>> u2.atoms.masses[:10]
-    array([2.03399992, 5.        , 5.        , 5.        , 8.01799965,
-       5.        , 0.034     , 5.        , 5.        , 5.        ])
-
-.. _`HMassRepartition`: http://parmed.github.io/ParmEd/html/parmed.html#hmassrepartition
-
-
-
-Classes
--------
-
-.. autoclass:: ParmEdReader
-   :members:
-
-.. autoclass:: ParmEdConverter
-   :members:
-
-
-"""
-import functools
-import itertools
 import warnings
+from ..converters.ParmEd import (ParmEdConverter, ParmEdReader,
```

Comment:
Added extra things to the import just in case someone used the underlying helper functions -- thoughts @lilyminium ?

(same thing for ParmedParser)

Reply:
Sure, don't see the harm